docs/source/examples/substrafl/go_further/run_iris_sklearn.ipynb

```diff
@@ -197,9 +197,9 @@
         "import numpy as np\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #94.        def predict(self, datasamples, shared_state):
</pre><p>is this function required as an implementation of an abstract method? </p><p>If so, I think it warrants an explanation</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Substra/substra-documentation/pull/376/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
Good catch thanks